dev/breeze/src/airflow_breeze/build_image/prod/build_prod_params.py

```diff
@@ -202,6 +214,15 @@ def extra_docker_build_flags(self) -> List[str]:
                     f"AIRFLOW_CONSTRAINTS_REFERENCE={self.default_constraints_branch}",
                 ]
             )
+        extra_build_flags.extend(
+            [
+                "--label",
```

Comment:
What do you think aboutt adding other labels like `io.artifacthub.package.maintainers`, `io.artifacthub.package.logo-url`?

Reply:
Sure. We can do it - care for adding them :) ?

Reply:
Will do.

Reply:
Done